docs/website/docs/SanityChecks.md

```diff
@@ -0,0 +1,18 @@
+# Sanity checks 
+
+This document is intended for anyone testing or benchmarking the Consensus component  outside this repository, for instance `cardano-node` or `cardano-node-tests`. We describe a series of sanity check that can be performed to rule out some known problems. 
+
+
+## Enable assertions 
+
+Even if the Consensus code is being benchmarked, it might be a good idea to *temporarily* enable assertions to perform additional checks on consistency.
+The following snippet can be added to the `cabal.project` file to enable assertions in our two main components:
+
+```cabal
+package ouroboros-consensus
+  ghc-options: -fno-ignore-asserts
```

Comment:
Isn't `flags: +asserts` the same?

Reply:
`program-options` won't work if invoked from the node's `cabal.project` though @jorisdral

Reply:
Thank you guys, I didn't know that we could use `+asserts` instead.